pkg/deploy/controller/deployerpod/controller.go

```diff
@@ -142,16 +136,18 @@ func (c *DeployerPodController) Handle(pod *kapi.Pod) error {
 			// If we are unable to get the deployment config, then the deploymentconfig controller will
 			// perform its duties once the resync interval forces the deploymentconfig to be reconciled.
 			name := deployutil.DeploymentConfigNameFor(deployment)
-			if config, err := c.client.DeploymentConfigs(deployment.Namespace).Get(name); err == nil {
+			kclient.RetryOnConflict(kclient.DefaultRetry, func() error {
```

Comment:
@ironcladlou updated to re-fetch. Note that I am deliberately ignoring any error here (other than conflicts) as the dc controller will eventually resync the config.